package.json

```diff
@@ -33,10 +33,8 @@
     "dependencies": "node ./scripts/postinstall.js"
   },
   "peerDependencies": {
-    "@sap/cds": "^8.0.0"
-  },
-  "dependencies": {
-    "@types/express": "^4.17.21"
+    "@sap/cds": "^8.0.0",
+    "@types/express": ">=5"
```

Comment:
@daogrady 
Just curious: why did you change `>=4` to `>=5`? Now the changelog entry is not correct too.

Reply:
You're right, that's obviously incorrect, thanks for pointing it out!